### ISSUE TITLE
dimensions of encoder_attention_mask in inputs of OFADecoder

### ISSUE THREAD

**Issue by SilyRab** [OFA-Sys/OFA#384]
This annotation may be wrong：
`encoder_attention_mask (`torch.Tensor` of shape `(bsz, seq_len)`): the padding mask of the source side.`
Does it refer to:
`shape (bsz, 1, tgt_len, src_len)`

**Comment by logicwong** [comment]
@SilyRab Do you mean `encoder_padding_mask`? If so, its shape is equal to (bsz, seq_len), as shown in [here](https://github.com/OFA-Sys/OFA/blob/5d3830fafa44bae0bf2e5d63f8b0463528034251/models/ofa/unify_transformer.py#LL980C1-L988C57)
